install_config/install/prerequisites.adoc

```diff
@@ -113,19 +113,19 @@ production environments, the following recommendations apply:
 
 Master Hosts::
 In a highly available {product-title} cluster with external etcd, a master host
-should have 1 CPU core and 1.5 GB of memory, on top of the defaults in the table
-above, for each 1000 pods. Therefore, the recommended size of master host in an
-{product-title} cluster of 2000 pods would be 2 CPU cores and 3 GB of RAM, in
-addition to the minimum requirements for a master host of 2 CPU cores and 16 GB
-of RAM.
+should have, in addition to the minimum requirements in the table above, 1 CPU
+core and 1.5 GB of memory for each 1000 pods. Therefore, the recommended size of
+a master host in an {product-title} cluster of 2000 pods would be the minimum
+requirements of 2 CPU cores and 16 GB of RAM, plus 2 CPU cores and 3 GB of RAM,
+totaling 4 CPU cores and 19 GB of RAM.
```

Comment:
@ahardin-rh just a note: the numbers should essentially be the same as before. The wording has changed to clarify the total requirements.